cdc/tests/integration_tests/disk_full/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=tikv-cdc.test
+SINK_TYPE=$1
+UP_PD=http://$UP_PD_HOST_1:$UP_PD_PORT_1
+DOWN_PD=http://$DOWN_PD_HOST:$DOWN_PD_PORT
+CF_ID="disk-full"
+
+function run() {
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
+	start_tidb_cluster --workdir $WORK_DIR
+	cd $WORK_DIR
+
+	cat - >"$WORK_DIR/tikv-cdc-config.toml" <<EOF
+per-changefeed-memory-quota=10485760 #10M
+[sorter]
+max-memory-consumption=0
+EOF
+	start_ts=$(get_start_ts $UP_PD)
+	export GO_FAILPOINTS='github.com/tikv/migration/cdc/cdc/sorter/unified/SorterFlushDiskFull=return(true)'
+	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --config $WORK_DIR/tikv-cdc-config.toml
+
+	case $SINK_TYPE in
+	tikv) SINK_URI="tikv://${DOWN_PD_HOST}:${DOWN_PD_PORT}" ;;
+	*) SINK_URI="" ;;
+	esac
+
+	tikv-cdc cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI" --changefeed-id=$CF_ID
+
+	rawkv_op $UP_PD put 5000
+
+	sleep 3
+	state=$(tikv-cdc cli changefeed list --pd=$UP_PD | jq .[0]."summary" | jq ."state" | tr -d '"')
+	if [[ "$state" == "normal" ]]; then
+		exit 1
+	fi
+
+	pid=$(ps -aux | grep "tikv-cdc" | awk '{print $2}' | head -n1)
```

Comment:
Suggest to use `pgrep -f "tikv-cdc"`. `ps aux | grep "tikv-cdc"` will have a change to get the pid of `grep`.

Reply:
Good suggestion, get it! Thank!